### PR TITLE
feat: uninstall app

### PR DESCRIPTION
**Pull request by itsspriyansh**
### Description

This PR adds the functionality to uninstall an app from an AVD or a real device. This feature can be accessed witht the following command: 
```bash
npx @nightwatch/mobile-helper android uninstall --app [--deviceId <deviceId>]
```

[Screencast from 02-08-24 11:44:40 AM IST.webm](https://github.com/user-attachments/assets/13008b80-9a13-4b84-a6e7-d1ee3fa783ca)

### CODE REVIEW

File: src/commands/android/subcommands/uninstall/app.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    Logger.log(`Uninstalling ${colors.cyan(packageName)}...\n`);
+
+    const uninstallationStatus = await execBinaryAsync(adbLocation, 'adb', platform, `-s ${options.deviceId} uninstall ${packageName}`);
+    if (uninstallationStatus?.includes('Success')) {
+      Logger.log(`${colors.green('App uninstalled successfully!')}\n`);
+
+      return true;
+    }
```
Comment (garg3133) [inline src/commands/android/subcommands/uninstall/app.ts:107]: `else` clause here similar to what we did in `uninstall --avd` PR.

File: src/commands/android/subcommands/uninstall/index.ts [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import {Options, Platform} from '../../interfaces';
+import {uninstallApp} from './app';
 import {deleteAvd} from './avd';
 
 export async function uninstall(options: Options, sdkRoot: string, platform: Platform): Promise<boolean> {
-  if (options.avd) {
+  if (options.app) {
+    return await uninstallApp(options, sdkRoot, platform);
+  } else if (options.avd) {
```
Comment (garg3133) [inline src/commands/android/subcommands/uninstall/index.ts:8]: We should use the `checkOptions` function here.

**Review by garg3133 — COMMENTED** [review]
`AVAILABLE_SUBCOMMANDS` should now contain all the subcommands and respective flags to be able to run the commands.

**Comment by itsspriyansh** [comment]
@garg3133  PR updated.

**Pull request merged**